vtysh/bgp_vty.c

```diff
@@ -3451,7 +3453,22 @@ cli_neighbor_set_peer_group_cmd_execute(char *vrf_name, const char *ip_addr,
         }
     }
 
-/* If peer group has a remote-as, it becomes primary. */
+    /* Synchronize peer timers with group timers. */
+    key_timers[0] = OVSDB_BGP_TIMER_KEEPALIVE;
+    key_timers[1] = OVSDB_BGP_TIMER_HOLDTIME;
+
+    if (!ovs_bgp_peer_group->n_timers) {
+    /* If peer-group timers are default */
+        tim_val.keepalive = 0;
+        tim_val.holdtime = 0;
+    } else {
+        memcpy(&tim_val, ovs_bgp_peer_group->value_timers, sizeof(tim_val));
```

Comment:
If size of tim_val is chaged someday, we can bump into memory corruption (if sizeof(tim_val <2 * sizeof(int64_t)))
Shouldn't change such way.